Makefile

```diff
@@ -49,18 +49,23 @@ sbt-test-transformer: .docker/sbt_test
 		--file=builds/sbt_image_builder.Dockerfile
 
 sbt-build-api: .docker/sbt_image_builder
+	TARGET=catatlogue_api/api \
```

Comment:
I’d be shocked if this is working like you expect, "catatlogue" vs. "catalogue".